api-report/sequence.api.md

```diff
@@ -92,7 +92,7 @@ export class Interval implements ISerializableInterval {
     // (undocumented)
     addPropertySet(props: PropertySet): void;
     // (undocumented)
-    auxProps: PropertySet[];
+    auxProps: PropertySet[] | undefined;
```

Comment:
Reviewers: are the changes in this file considered breaking? They could theoretically cause code relying on them to stop compiling, but they are undocumented, and the breaking-change checks are not failing.